src/components/calculator.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './calculator.css';
+import calculate from '../logic/calculate';
+import '../App.css';
+
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+
+  const handleEvent = (e) => {
+    setState({ ...state, ...calculate(state, e.target.name) });
+  };
+
+  return (
+    <div className="calculatorSection">
+      <h1>Let&#39;s do some math!</h1>
+      <div className="calc">
+        <p className="result">
+          <span>{state.total}</span>
+          <span>{state.operation}</span>
+          <span>{state.next}</span>
+        </p>
+        <div className="calculator">
+          <button type="button" className="grayButton" name="AC" onClick={handleEvent}>AC</button>
+          <button type="button" className="grayButton" name="+/-" onClick={handleEvent}>+/-</button>
+          <button type="button" className="grayButton" name="%" onClick={handleEvent}>% </button>
+          <button type="button" className="orangeButton" name="÷" onClick={handleEvent}>÷</button>
+          <button type="button" className="grayButton" name="7" onClick={handleEvent}>7</button>
+          <button type="button" className="grayButton" name="8" onClick={handleEvent}>8</button>
+          <button type="button" className="grayButton" name="9" onClick={handleEvent}>9</button>
+          <button type="button" className="orangeButton" name="x" onClick={handleEvent}>x</button>
+          <button type="button" className="grayButton" name="4" onClick={handleEvent}>4</button>
+          <button type="button" className="grayButton" name="5" onClick={handleEvent}>5</button>
+          <button type="button" className="grayButton" name="6" onClick={handleEvent}>6</button>
+          <button type="button" className="orangeButton" name="-" onClick={handleEvent}>-</button>
+          <button type="button" className="grayButton" name="1" onClick={handleEvent}>1</button>
+          <button type="button" className="grayButton" name="2" onClick={handleEvent}>2</button>
+          <button type="button" className="grayButton" name="3" onClick={handleEvent}>3</button>
+          <button type="button" className="orangeButton" name="+" onClick={handleEvent}>+</button>
+          <button type="button" className="grayButton-0" name="0" onClick={handleEvent}>0</button>
+          <button type="button" className="grayButton" name="." onClick={handleEvent}>.</button>
+          <button type="button" className="orangeButton" name="=" onClick={handleEvent}>=</button>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
This page doesn't match the wireframe design.

<img width="452" alt="Screen Shot 2022-08-17 at 8 06 45" src="https://user-images.githubusercontent.com/82426682/185141890-07fef41f-f019-44f1-89ef-6e0df9deff96.png">

<img width="1440" alt="Screen Shot 2022-08-17 at 8 07 13" src="https://user-images.githubusercontent.com/82426682/185141986-9b577740-a015-4935-b7a2-c3515b8a744d.png">

Please fix it.